extension/package.json

```diff
@@ -17,7 +17,7 @@
     "check": "npm run lint && prettier --check .",
     "doctoc": "doctoc . --github",
     "format": "prettier --write .",
-    "zip-google-function": "cp index.googleFunction.js index.js && zip -r extension-module.zip src resources index.js package.json && rm index.js",
+    "zip-google-function": "cp index.googleFunction.js index.js && zip -r extension-module.zip src resources .extensionrc index.js package.json && rm index.js",
     "zip-lambda-function": "cp index.lambda.js index.js && npm ci --production && zip -r extension-module.zip . && rm index.js",
```

Comment:
What about external file for lambda?

Reply:
for lambda archive is zipping all content in the current folder, google function is picking only the needed parts, that's why there is a addition for the google script.